conductor/lib/uploader_v2.py

```diff
@@ -281,15 +280,34 @@ def __init__(self,
         self.last_touch = datetime.datetime.now()
         self._results_queue = results_queue
 
+    def log(self, level, message, fields={}):
+        """
+        Create a formatted log message from passed in data, automatically
+        add process info, and call LOGGER with result.
+        """
+        fields["process_id"] = self._id
+        fields["process_state"] = self._run_state.value
+        if self.current_upload:
+            fields["filepath"] = self.current_upload.get("filepath")
+            fields["upload_id"] = self.current_upload.get("upload_id")
+            fields["md5"] = self.current_upload.get("md5")
+        log_msg = functools.reduce(item_formatter, fields.items(), message)
+        log_fun = getattr(LOGGER, level, "warning")
+        log_fun(log_msg)
+        return
+
     def run(self):
         # Set the run_state value to "running"
         self._run_state.value = Uploader.STATE_RUNNING
 
+        self.log("info", "STARTING_UPLOAD_PROCESS")
```

Comment:
instead of using an `"info"` string, perhaps using `logging.INFO` (or at least some decrared constant)
With the current implementation of the `UploaderWorker.log` method, if the `"info"` string was mistakenly spelled as `"ifno"`, the message would be logged as "warning" level message (and  no exception would indicate that there's something wrong with the code/implementation).

Reply:
If you were to use the suggested changes for the `UploaderWorker.log` method, 
This call would then look like
`self.log("STARTING_UPLOAD_PROCESS")`
or if wanting to be more explicit:
`self.log("STARTING_UPLOAD_PROCESS", level=logging.INFO)`

Reply:
I thought about doing exactly that, but then I decided that I liked having the log level at the front of the call. Using the logging constants makes sense.